test/e2e/kubectl/portforward.go

```diff
@@ -492,6 +657,18 @@ var _ = SIGDescribe("Kubectl Port forwarding", func() {
 		ginkgo.It("should support forwarding over websockets", func(ctx context.Context) {
 			doTestOverWebSockets(ctx, "localhost", f)
 		})
+
+		ginkgo.It("should support server never read request, and disconnects", func(ctx context.Context) {
+			e2eskipper.Skipf("Ref: https://issues.k8s.io/74551 dependency on the container runtime")
+			doTestConnectionNeverReadRequest(ctx, "localhost", f)
+		})
+	})
+
+	ginkgo.Describe("with a server that sends RST upon accepting a connection", func() {
+		ginkgo.It("should connect, send data, and then connect again", func(ctx context.Context) {
+			e2eskipper.Skipf("Ref: https://issues.k8s.io/74551 dependency on the container runtime")
```

Comment:
this test should work with your fix, we should not skip it

Reply:
I think the reason this is skipped is that it requires a new version of containerd with https://github.com/containerd/containerd/pull/8418, which has yet to be merged.

When I tested, I built and used a version of containerd with that fix *in addition to* the fix in this PR and *both* PRs were required in order for this e2e test to pass.

Reply:
> Perhaps we should clarify a bit here.

Our goal here is to deal with [EPIPE, ECONNRESET] reasonably, and these two related e2e tests need to rely on [Containerd's PR](https://github.com/containerd/containerd/pull/8418) to be merged.

Reply:
can you do a temporal commit to remove the skip here so we can see the result on the CI?
 I think that this should work without the containerd changes, but I may be wrong
I would test it myself but the CI is faster and I'm a bit busy :)

Reply:
PTAL

Reply:
ok, thanks for doing the exercise, appreciate it